From: Joel Brobecker <brobecker@gnat.com>
Date: Fri, 19 Aug 2011 15:51:10 +0000 (+0000)
Subject: Include the <root>/cpu/ dir in GDB releases
X-Git-Url: http://sourceware.org/git/?p=gdb.git;a=commitdiff_plain;h=5970c60a5c4b9440686f9c60cda5268b6e80a910

Include the <root>/cpu/ dir in GDB releases

This directory contains some files that are used to generate some
of the files included in the GDB release, and thus must be part of
the GDB release also.

ChangeLog:

        * src-release (GDB_SUPPORT_DIRS): Add 'cpu'.
---

Index: gdb-7.3/ChangeLog
===================================================================
--- gdb-7.3.orig/ChangeLog	2011-08-23 21:40:16.383403574 +0100
+++ gdb-7.3/ChangeLog	2011-08-23 21:41:01.771403554 +0100
@@ -1,3 +1,7 @@
+2011-08-19  Joel Brobecker  <brobecker@adacore.com>
+
+	* src-release (GDB_SUPPORT_DIRS): Add 'cpu'.
+
 2011-05-17  Joseph Myers  <joseph@codesourcery.com>
 
 	* configure.ac (avr-*-*): Add comment about why libssp is disabled.
Index: gdb-7.3/src-release
===================================================================
--- gdb-7.3.orig/src-release	2011-08-23 21:40:16.375403574 +0100
+++ gdb-7.3/src-release	2011-08-23 21:41:01.771403554 +0100
@@ -269,7 +269,7 @@
 		SUPPORT_FILES="$(GNATS_SUPPORT_DIRS)"
 
 .PHONY: gdb.tar.bz2
-GDB_SUPPORT_DIRS= bfd include libiberty opcodes readline sim intl libdecnumber
+GDB_SUPPORT_DIRS= bfd include libiberty opcodes readline sim intl libdecnumber cpu
 gdb.tar.bz2: $(DIST_SUPPORT) $(GDB_SUPPORT_DIRS) gdb
 	$(MAKE) -f $(SELF) gdb-taz TOOL=gdb \
 		MD5PROG="$(MD5PROG)" \
